decidim-core/app/cells/decidim/attachments_upload_tab_cell.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Decidim
+  class AttachmentsUploadTabCell < Decidim::ViewModel
+    alias form model
+
+    def show
+      render :show
+    end
```

Comment:
I think this method is not required, the default behavior is to render the show template

Reply:
https://github.com/decidim/decidim/pull/12917#discussion_r1630969029